### ISSUE TITLE
Can't install using easy_install or PIP

### ISSUE THREAD

**Issue by baltasvejas** [pytube/pytube#14]
First of all thanks for maintaining such great tool.
I hope this is temporary issue, but I can't install it either using 
sudo pip install pytube
either
sudo easy_install pytube

I tried on ubuntu 13.10 and ubuntu 13.04

me@ubuntu:~$ sudo easy_install pytube
Searching for pytube
Reading http://pypi.python.org/simple/pytube/
Couldn't find index page for 'pytube' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading http://pypi.python.org/simple/
No local packages or download links found for pytube
error: Could not find suitable distribution for Requirement.parse('pytube')
me@ubuntu:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 13.10
Release:        13.10
Codename:       saucy
me@ubuntu

me@ubuntu:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 13.10
Release:        13.10
Codename:       saucy

me@ubuntu:~$ sudo pip install pytube
Downloading/unpacking pytube
  Real name of requirement pytube is pytube
  Could not find any downloads that satisfy the requirement pytube
Cleaning up...
No distributions at all found for pytube
Storing complete log in /home/tv/.pip/pip.log
me@ubuntu:~$

**Comment by nficano** [comment]
I am on this sir. I'll have this fixed tonight.

**Comment by shshank** [comment]
Hi,
Was this issue resolved?
I tried to install pytube with pip and easy install. on OSX 10.6.8, python 2.7.5

## With pip 1.4.1:

```
$ pip install pytube
Downloading/unpacking pytube
Real name of requirement pytube is pytube
Could not find any downloads that satisfy the requirement pytube
Cleaning up...
No distributions at all found for pytube
Storing complete log in /Users/Shashank/.pip/pip.log
```

## With easy_install (setuptools 2.0): 

```
$ easy_install pytube
Searching for pytube
Reading https://pypi.python.org/simple/pytube/
Couldn't find index page for 'pytube' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
No local packages or download links found for pytube
error: Could not find suitable distribution for Requirement.parse('pytube')
```

I installed it by `pip install https://github.com/NFicano/pytube/archive/master.zip`

**Comment by baltasvejas** [comment]
Hi, I think still doesn't work...
I use pip install youtube-dl. This is another tool, which solved my issue.
My pip freeze:
Django==1.6
MySQL-python==1.2.4
Pillow==2.2.1
South==0.8.4
argparse==1.2.1
django-sslify==0.2
wsgiref==0.1.2
youtube-dl==2014.01.03

**Comment by arudmin** [comment]
The same problem:

(venv)a@vl:~/code/yt2in$ sudo easy_install pytube
Searching for pytube
Reading https://pypi.python.org/simple/pytube/
Couldn't find index page for 'pytube' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
No local packages or download links found for pytube
error: Could not find suitable distribution for Requirement.parse('pytube')
(venv)a@vl:~/code/yt2in$ pip search pytube
(venv)a@vl:~/code/yt2in$ pip install pytube
Downloading/unpacking pytube
  Real name of requirement pytube is pytube
  Could not find any downloads that satisfy the requirement pytube
Cleaning up...
No distributions at all found for pytube
Storing debug log for failure in /home/odmin/.pip/pip.log

**Comment by damiencorpataux** [comment]
Same here, please give us some time to package that tool for NFicano

It's easy to install it from source though:

``` bash
git clone https://github.com/NFicano/pytube.git
cd pytube
sudo python setup.py install
```

or, as [mentioned](https://github.com/NFicano/pytube/issues/14#issuecomment-31574463) by @shshank:

```
[sudo] pip install https://github.com/NFicano/pytube/archive/master.zip
```

**Comment by josegonzalez** [comment]
@NFicano do you mind if I push releases of this to pypi?

**Comment by nficano** [comment]
not at all :+1:

**Comment by josegonzalez** [comment]
Mind adding me as a contributor to this repository so I can make any necessary changes?

**Comment by nficano** [comment]
done

**Comment by josegonzalez** [comment]
@NFicano did you register the package on pypi before? It seems I cannot create releases because the package is registered.

**Comment by nficano** [comment]
what's your pypi username, I'll add you as a maintainer.

**Comment by josegonzalez** [comment]
It is `savant`

**Comment by nficano** [comment]
all set

On Saturday, September 27, 2014, Jose Diaz-Gonzalez <
notifications@github.com> wrote:

> It is savant
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/NFicano/pytube/issues/14#issuecomment-57073177.

## 

Sent from my iPhone.

**Comment by josegonzalez** [comment]
I have released version 0.2.0. Let me know if you have trouble with it!
